iroha-cli/CMakeLists.txt

```diff
@@ -61,7 +61,6 @@ target_link_libraries(iroha-cli
     client
     cli-flags_validators
     keys_manager
-    boost
```

Comment:
How are these boost changes related to pull request description?

Reply:
I will remove them. Actually all these boost targets are not needed, everything works without them.